handbook/fasse.md

```diff
@@ -194,40 +194,51 @@ data/
 
 ## Step 8. Scratch Space
 
-Scratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. The high-performance storage and resource isolation capabilities of a scratch space enhance performance, streamline workflows, and protect valuable data. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
+A acratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
```

Comment:
'scratch' in first line misspelled. could link to FAS RC's scratch storage policy doc: https://docs.rc.fas.harvard.edu/kb/policy-scratch/

Reply:
maybe note data store in scratch will be deleted after 90 days

Reply:
this is partially explained in the above, but I believe the scratch datasystem is highly optimized for high-throughput file read/write. this means if you have to do lots of edits to large files or work with many small files, scratch may provide a more efficient means of operating.